modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServiceDeployerApi.java

```diff
@@ -354,14 +347,8 @@ public ResponseEntity<List<String>> getAvailabilityZones(
         UUID uuid = StringUtils.isBlank(serviceId) ? null : UUID.fromString(serviceId);
         String currentUserId = this.userServiceHelper.getCurrentUserId();
         OrchestratorPlugin orchestratorPlugin = pluginManager.getOrchestratorPlugin(csp);
-        List<String> availabilityZones = orchestratorPlugin.getAvailabilityZonesOfRegion(
+        return orchestratorPlugin.getAvailabilityZonesOfRegion(
```

Comment:
@baixinsui instead of completely returning CacheControl, can we still return it when we know that the response has no errors?
Advantage is, if we add cacheControl then the browser will send these requests again. (Response will be additionally cached in browser)

Reply:
Updated. Remove the cache control when catch exceptions